### DISCUSSION TITLE
T2i_adapter Color in Comfyui

### DISCUSSION THREAD

**Discussion by achuntai** [Fannovel16/comfyui_controlnet_aux#397] [Q&A]
Hello everyone, I hope you are well. I wanted to ask if you could tell me which nodes I should consider to load the preprocessor and the T2i Adapter Color model. I'm learning ComfyUI so it's a bit difficult for me. On the one hand I found the "Color Palette" preprocessor loader and connected it to the "Apply ControlNet (Advance)" node like this:

![image](https://github.com/Fannovel16/comfyui_controlnet_aux/assets/148453128/dc9b10d5-cb38-44d9-a2cb-c4c1fc760cf7)

I have tried different T2i Adapter Color preprocessors but none of them have worked for me:
![image](https://github.com/Fannovel16/comfyui_controlnet_aux/assets/148453128/6498ba68-71fd-46bb-a267-c58bbff3787a)

but when I click on "Queue Prompt" I get the following error:

> Error occurred when executing ControlNetApply:
> 
> 'NoneType' object has no attribute 'copy'
> 
> File "C:\Users\Se_RC\AppData\Roaming\StabilityMatrix\Packages\ComfyUI\execution.py", line 151, in recursive_execute
> output_data, output_ui = get_output_data(obj, input_data_all)
> File "C:\Users\Se_RC\AppData\Roaming\StabilityMatrix\Packages\ComfyUI\execution.py", line 81, in get_output_data
> return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
> File "C:\Users\Se_RC\AppData\Roaming\StabilityMatrix\Packages\ComfyUI\execution.py", line 74, in map_node_over_list
> results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
> File "C:\Users\Se_RC\AppData\Roaming\StabilityMatrix\Packages\ComfyUI\nodes.py", line 761, in apply_controlnet
> c_net = control_net.copy().set_cond_hint(control_hint, strength)

If anyone could help me, I would be super grateful.

Hasta pronto, gracias!

**Comment by muneeb2001** [comment]
Did you find any solution?

**Reply by achuntai** [reply]
No mate, after some time after having consulted I have returned to this post but unfortunately not
